### ISSUE TITLE
Backup Schedule

### ISSUE THREAD

**Issue by seularts** [keepassxreboot/keepassxc#6180]
KeePassXC is in dire need for a better backup feature. It would be nice to have an option that allows the user to create the backup when he closes the application and not ONLY after he makes a save that keep rewriting itself. In addition, the user should have the ability to define how many backup versions he wants. For example: If I make 5 modifications during a day (and close the app 5 times) I would like to have these 5 individual backups (depending on the number of backups that the user defines in the settings, ex: 7). This ensures that the database can be recovered from a previous date in case it is corrupted. I am aware that I can do this manually, but I am looking for productivity in a software, not feature roadblocks.

Thank you for your time :)

**Comment by droidmonkey** [comment]
Discussed: #3279 #6014 #5259 #4603

**Comment by seularts** [comment]
Yep, those guys are right. This is an imperative feature that needs to exist for better consistence :) Glad I am not alone <3
